package.json

```diff
@@ -40,10 +40,11 @@
         "code-pre-tests": "ts-mocha --no-timeouts ./test/*.ts",
         "test": "ts-mocha ./@here/*/test/*.ts",
         "performance-test-node": "ts-mocha ./test/performance/**/*.ts",
+        "cleanup": "./scripts/cleanup.sh",
         "cov-test": "nyc mocha -r ts-node/register ./@here/*/test/*.ts",
         "cov-report-html": "nyc report --reporter=html",
         "start": "ts-node ./scripts/credentials.ts -- . && webpack-dev-server -d --config @here/harp-examples/webpack.config.js",
-        "build": "npm run build-bundle && npm run build-examples",
+        "build": "npm run cleanup && npm run build-bundle && npm run build-examples",
```

Comment:
I would keep the clean and the build step separate and call the clean before the build just in travis.